data/data/cluster-api/ibmcloud-infrastructure-components.yaml

```diff
@@ -4325,7 +5017,7 @@ spec:
         env:
         - name: IBM_CREDENTIALS_FILE
           value: /home/.ibmcloud/ibm-credentials.env
-        image: gcr.io/k8s-staging-capi-ibmcloud/cluster-api-ibmcloud-controller:v0.8.0-alpha.0
+        image: gcr.io/k8s-staging-capi-ibmcloud/cluster-api-ibmcloud-controller:v0.8.0
```

Comment:
can you get the this yaml directly from the Assets link from the release - https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/releases/tag/v0.8.0, this image should be pointing to the registry.k8s.io/capi-ibmcloud/cluster-api-ibmcloud-controller:v0.8.0

Reply:
I did
```
make release-manifests
```
on
```
hamzy:cluster-api-provider-ibmcloud$ git show HEAD
commit 8cd12b5a51def07c4c11d6d83eb0bd298be67240 (HEAD -> main, tag: v0.8.0, origin/main, origin/HEAD)
```
Why doesn't that work?

Reply:
I think we need to fix the image name for that makefile target, its safe to use the image the image pushed in the `registry.k8s.io` image registry.

Reply:
```
hamzy:hamzy-installer$ grep 'image: ' ./data/data/cluster-api/ibmcloud-infrastructure-components.yaml
        image: gcr.io/k8s-staging-capi-ibmcloud/cluster-api-ibmcloud-controller:v0.8.0
        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0
hamzy:hamzy-installer$ curl --silent --location --output - https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/releases/download/v0.8.0/infrastructure-components.yaml | grep 'image: '
        image: registry.k8s.io/capi-ibmcloud/cluster-api-ibmcloud-controller:v0.8.0
        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0
```